scripts/biomedical/chembl/test-data/chembl_29_bio_expected.csv

```diff
@@ -0,0 +1,10 @@
+Sequence,chembl,name,dcid
```

Comment:
Underscores should be replaced by spaces in the name. Quotes should be around text values